cli/src/commands/install.js

```diff
@@ -417,35 +417,39 @@ async function installNpmLibDef(
   // Find the libDef in the cached repo
   try {
     const terseFilePath = path.relative(
-      path.resolve(npmDir, '..', '..'),
-      filePath,
+      path.resolve(npmDir, "..", ".."),
+      filePath
     );
 
     if (!overwrite && (await fs.exists(filePath))) {
       console.error(
-        '  • %s\n' + '    └> %s',
+        "  • %s\n" + "    %s\n    %s\n    └> %s",
         colors.bold(
           colors.red(
             `${terseFilePath} already exists and appears to have been manually ` +
-              `written or changed!`,
-          ),
+              `written or changed!`
+          )
+        ),
+        colors.green(
```

Comment:
My changes are this and next line. I think flow-typed prettier is not maybe in best possible shape :D